src/property/types/CloudString.h

```diff
@@ -49,6 +49,12 @@ class CloudString : public Property {
     void clear() {
       _value = PropertyActions::CLEAR;
     }
+    const char * c_str() {
+      return _value.c_str();
+    }
+    String& getValue() {
```

Comment:
Maybe a const reference `String const &` ? Otherwise you could and up with a unplanned modification of the underlying string.

Reply:
this change was dropped